nvm.sh

```diff
@@ -2407,6 +2426,7 @@ nvm() {
       nvm_echo '    --lts                                     When listing, only select from LTS (long-term support) versions'
       nvm_echo '    --lts=<LTS name>                          When listing, only select from versions for a specific LTS line'
       nvm_echo '  nvm deactivate                              Undo effects of `nvm` on current shell'
+      nvm_echo '    --silent                                  Run nvm deactivate in silent mode'
```

Comment:
```suggestion
      nvm_echo '  nvm deactivate [--silent]                   Undo effects of `nvm` on current shell'
```
for consistency with every other command that has `--silent`